frontend/src/layouts/Admin.tsx

```diff
@@ -43,6 +43,11 @@ function AdminLayout(props: LayoutProps) {
               <FontAwesomeIcon icon={faWindowClose} size="lg" role="img" />
             </button>
             <ul className="usa-nav__primary usa-accordion">
+              <li className="usa-nav__primary-item">
+                <a className="usa-nav__link" href="/admin/settings/topicarea">
+                  Settings
+                </a>
```

Comment:
This should be using the `<Link />` component instead so that it properly navigates using react router.

Reply:
You can either use our own `<Link />` component or the one in `react-router-dom`.